pset-functions.js

```diff
@@ -75,6 +91,64 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+// const score  = {
+//     e:1, a:1, i:1, o:1, u:1, l:1, n:1, r:1, s:1, t:1,
+//     d:2, g:2,
+//     b:3, c:3, m:3, p:3,
+//     f:4, h:4, v:4, w:4, y:4,
+//     k:5,
+//     j:8, x:8,
+//     q:10, z:10
+// }
+//
+// function scrabbleScore(word) {
+//   let total = 0
+//   for (i = 0; i <= word.length; i++) {
+//     total = total + score[word[i]].values
+//   }
+//   return total
+// }
+//
+// console.log(scrabbleScore("cabbage"))
+
+const scrabbleWords  = {
+e: 1,
+a: 1,
+i: 1,
+o: 1,
+u: 1,
+l: 1,
+n: 1,
+r: 1,
+s: 1,
+t: 1,
+d: 2,
+g: 2,
+b: 3,
+c: 3,
+m: 3,
+p: 3,
+f: 4,
+h: 4,
+v: 4,
+w: 4,
+y: 4,
+k: 5,
+j: 8,
+x: 8,
+q: 10,
+z: 10
+}
+function scrabbleScore(word) {
+let score = 0;
+for (let i = 0; i < word.length; i++) {
+    score = score + scrabbleWords[word[i]]
```

Comment:
Really great use of navigating through an object and using it's key/value pair in bracket notation to get sum the values! Just for your benefit and future reference, please find an alternate solution using forEach and .includes on an array [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L97). Also, be conscious of indentation; the block inside the function would be much easier to read if it were indented 2 space.